### PR TITLE
Leashes and Buzzers!~

### PR DESCRIPTION
**Pull request by GremlingSS**
### Leashes
These will be able to clip to people who have a collar worn (I'm going to add a check for just their neck and in their uniform's content)

Once a player has been "attacked" by a leash (clicked on), they will be linked to the "attacker" after a short timer, and is forced into a pull state, unable to move. Simply dropping the leash will unclip it automatically (muh immersions ruined!) Or if the attacker falls over/clicks on them again.

This is gonna take some time to test and make sure it's safe, hence why I'm not confident I can do it atm, but I'll try my best to get it to work.

### Buzzers (vibrators)
Just gonna make it so you can click to make the vibrators buzz, it'll emit a tiny sound which you can only hear in 1 tile range and a small italic writing of a *buzz* every now and again. c:

#### Checklist
 - [x] Leashes
 - [x] Buzzers

### CODE REVIEW

**Comment by GremlingSS** [comment]
It worksn't

Works perfectly fine with basic mobs, but the `if((typesof(/obj/item/clothing/accessory/collar) in HV.w_uniform.contents))` will not be true, it's always false, making it not work on humans.

I tried `if((/obj/item/clothing/accessory/collar in HV.w_uniform.contents))` too which didn't work. I need a hero, my puny midnight brain cannot think of what function BYOND uses to check for an atom plus it's subtypes in a list..!

**Comment by GremlingSS** [comment]
oop

**Comment by GremlingSS** [comment]
OH MY FUCKING GOD WHY IS GITHUB STILL SUCH A PAIN REEEE

File: code/__defines/_compile_options.dm [hunk 1]
```diff
@@ -19,3 +19,5 @@
 	#define USING_MAP_DATUM /datum/map
 	#define MAP_OVERRIDE 1
 #endif
+
```
Comment (DameonOwen) [inline code/__defines/_compile_options.dm:22]: Is this.... necessary?

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -846,6 +846,9 @@
 			lying = incapacitated(INCAPACITATION_KNOCKDOWN)
 			canmove = !incapacitated(INCAPACITATION_DISABLED)
 
+	if(leashed) // ARFS ADD: LEASHES BOIIIIIIIIIIIIIIIIIIIII
```
Comment (DameonOwen) [inline code/modules/mob/living/living.dm:849]: I know we really don't want to be touching base files much. @TalkingCactus Thoughts?

Comment (TalkingCactus) [reply code/modules/mob/living/living.dm:849]: DO NOT TOUCH BASE FILES

File: code/__defines/_compile_options.dm (hunk 1 above)
Comment (GremlingSS) [reply code/__defines/_compile_options.dm:22]: @DameonOwen 
honestly forgot how to revert a commit so I was like. *shrug

File: code/modules/mob/living/living.dm (hunk 2 above)
Comment (GremlingSS) [reply code/modules/mob/living/living.dm:849]: Only way I can implement it this way. I even said in voice.

**Comment by GremlingSS** [comment]
neato I can bother you guys to review it, hehe

**Review by DameonOwen — APPROVED** [review]

**Pull request merged**